src/hotspot/cpu/aarch64/foreign_globals_aarch64.cpp

```diff
@@ -1,6 +1,6 @@
 /*
- * Copyright (c) 2020, Oracle and/or its affiliates. All rights reserved.
- * Copyright (c) 2019, Arm Limited. All rights reserved.
+ * Copyright (c) 2020, 2021, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2019, 2021, Arm Limited. All rights reserved.
```

Comment:
Only update third-party copyrights under direction from that copyright holder. This may not be the correct format for example.
Also it is 2022. :)

Reply:
I think the Arm Ltd one was probably changed by me in one of the PRs in the Panama repo.

Reply:
Right, I brought in these commits from the Panama repo, so this is from Nick's updates there (specifically from this PR: https://github.com/openjdk/panama-foreign/pull/610).

I'm not sure what to do here. I own the PR, but Nick is a contributor on it, so some of the changes are his (including this copyright year change).

Reply:
That's fine, just needed to check. But as these are now being committed to mainline the year does need to change to 2022. But @nick-arm will have to confirm whether swapping 2022 for 2021 is correct in the ARM copyright, or whether it becomes e.g. "2019, 2021, 2022,".

Reply:
@nick-arm Thanks!